### DISCUSSION TITLE
Unable to access Microsoft Azure Student Credits

### DISCUSSION THREAD

**Discussion by karthithedoc** [community/community#146836] [GitHub Education]
### Hi everyone,

I have got verified GitHub students developer pack but i cant access the microsoft Azure free credits.
![image_2024-12-11_202725254](https://github.com/user-attachments/assets/289d5cfc-b9c6-4843-9fe4-dcc70edcb02e)

**Comment by davevad93** [comment]
Hi @karthithedoc ,

This has been solved here #135739 , you have the following options:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I recommend the first one.
